src/Estimators/src/SubModel.cpp

```diff
@@ -489,19 +498,73 @@ std::size_t RDE::SubModel::getNrOfExternalContact() const
     return m_externalContactList.size();
 }
 
-const RDE::FT& RDE::SubModel::getFTSensor(const int index) const
+const RDE::FT& RDE::SubModel::getFTSensor(const std::string& name)
+{
+    if (m_ftList.find(name) != m_ftList.end())
```

Comment:
Note that ``find`` already gives you the iterator to the object, if it has been found. I would suggest using its output if it is valid. The same holds below